docs/HowToUpgrade.md

```diff
@@ -22,39 +22,40 @@ The main resources installed in a single Kubernetes cluster for a single NuoDB d
 | Hot-copy Storage Manager (HCSM)  | database         | StatefulSet              | database | SMs with hot-copy operations enabled |
 | Storage Manager (SM)             | database         | StatefulSet              | database | SMs with hot-copy operations disabled |
 
-An example multi-cluster NuoDB deployment scenario can be seen in the diagram below.
+An example of a multi-cluster NuoDB deployment scenario is shown in the diagram below.
 
 ![NuoDB Deployment Example](../images/nuodb-deployment-example.png)
 
 This document focuses on upgrading NuoDB software and providing details on the upgrade mechanics specific to Kubernetes Deployments.
```

Comment:
Suggest: 

"This document describes the NuoDB software upgrade process and provides details ...."